raddb/dictionary.inverse

```diff
@@ -34,6 +34,7 @@ ATTRIBUTE       PacketFence-IfIndex                   19       string
 ATTRIBUTE       PacketFence-Profile                   20       string       
 ATTRIBUTE       PacketFence-Domain                    21       string
 ATTRIBUTE       PacketFence-Computer-Name             22       string
+ATTRIBUTE       PacketFence-Radius-Ip                 32       string
```

Comment:
Put this at the end of the list

Reply:
i have other pull request that add attributes in the dictionary, so it will stay as is until all the other will be merged.

Reply:
![image](https://user-images.githubusercontent.com/3857942/58097535-a6d04c80-7ba5-11e9-8ba7-fb009c6b9e4c.png)